overrides/app/pages/algolia-product-list/partials/algolia-range-refinements.jsx

```diff
@@ -28,66 +31,100 @@ const AlgoliaRangeRefinements = (props) => {
+            if (val < minValue) {
+                val = minValue
+            }
+            setMaxValue(val)
+            refine([minValue, val])
+        }, 500)
+        setMaxTimeout(newTimeout)
+    }
+
     return (
-        <AlgoliaRefinementsContainer title={props.title} attributes={[props.attribute]}>
-            <Box pb="1">
-                <RangeSlider
-                    aria-label={['min', 'max']}
-                    defaultValue={start}
-                    min={range.min}
-                    max={range.max}
-                    step={step}
-                    onChange={(val) => {
-                        setSliderValue(val)
-                        refine(val)
-                    }}
-                >
-                    <RangeSliderMark
-                        value={sliderValue[0]}
-                        textAlign="center"
-                        bg="black"
-                        color="white"
-                        p="1"
-                        mt="0.5"
-                        fontSize="sm"
-                    >
-                        {intl.formatNumber(sliderValue[0], {
-                            ...fnOpts,
-                            currency
-                        })}
-                    </RangeSliderMark>
-                    <RangeSliderMark
-                        value={sliderValue[1]}
-                        textAlign="center"
-                        bg="black"
-                        color="white"
-                        p="1"
-                        mt="0.5"
-                        fontSize="sm"
-                        left={isMax && 'auto !important'}
-                        right={isMax && '0'}
-                    >
-                        {intl.formatNumber(sliderValue[1], {
-                            ...fnOpts,
-                            currency
-                        })}
-                    </RangeSliderMark>
-                    <RangeSliderTrack bg="gray.200">
-                        <RangeSliderFilledTrack bg="black" />
-                    </RangeSliderTrack>
-                    <RangeSliderThumb boxSize={6} index={0} border="2px" borderColor="gray.200" />
-                    <RangeSliderThumb boxSize={6} index={1} border="2px" borderColor="gray.200" />
-                </RangeSlider>
-            </Box>
-        </AlgoliaRefinementsContainer>
+        <AccordionItem>
+            <h2>
+                <AccordionButton>
+                    <Box as="span" flex="1" textAlign="left">
+                        {props.title}
+                    </Box>
+                    <AccordionIcon />
+                </AccordionButton>
+            </h2>
+            <AccordionPanel pb={4}>
+                <AlgoliaRefinementsContainer attributes={[props.attribute]}>
+                    <Box pb="1" display="flex" gap="4">
+                        <FormControl>
+                            <FormLabel htmlFor="minValue">Min</FormLabel>
+                            <Input
```

Comment:
Why are we re-doing everything instead of using Algolia's built-in `RangeInput`? https://www.algolia.com/doc/api-reference/widgets/range-input/react/

Reply:
We should tend to the desired design, but if there are small discrepancies it's fine in my opinion. The goal of the demo is to show Algolia components. If we find limitation on them, they should be fixed in our components directly.
Do you think it's possible to deploy 2 versions in parallel so we can see the differences?